libs/@blockprotocol/graph/src/types/subgraph/graph-resolve-depths.ts

```diff
@@ -1,11 +1,22 @@
+export interface OutgoingEdgeResolveDepth {
+  outgoing: number;
+}
+
 export interface EdgeResolveDepths {
   incoming: number;
   outgoing: number;
 }
 
 /** @todo - Add documentation */
-/** @todo - expand this with ontology - related edges */
 export type GraphResolveDepths = {
   hasLeftEntity: EdgeResolveDepths;
   hasRightEntity: EdgeResolveDepths;
+
+  // @todo decide if these should be required
```

Comment:
I think these should be required, but we can use `Partial` in function parameters.

For now, when we _create_ one of these complete objects for `Subgraph`s in `mock-block-dock` or otherwise we can just fill in a depth of `0` for the ones we don't support at the moment?

Reply:
Yep – done in https://github.com/blockprotocol/blockprotocol/pull/946/commits/12ecc76c552c4df3e6767f625425bf474b2dfbaa